Makefile

```
@@ -115,7 +164,8 @@ build: build-linux-amd build-linux-arm build-macos-intel build-macos-apple build
 
 #@ Lint
 define run-lint
-	@if command -v ${1} &> /dev/null; then \
+	@export PATH="$${PATH}:$${PWD}/.bin"; \
+	if command -v ${1} > /dev/null; then \
```

Should be `&>` (equivalent to `2>&1`) to suppress both stdout and stderr. We only care about the exit code

```suggestion
	if command -v ${1} &> /dev/null; then \
```